doc/source/howtos/proxysql-v2.rst

```diff
@@ -0,0 +1,816 @@
+
+   # proxysql admin interface credentials.
+   export PROXYSQL_DATADIR='/var/lib/proxysql'
+   export PROXYSQL_USERNAME='admin'
+   export PROXYSQL_PASSWORD='admin'
+   export PROXYSQL_HOSTNAME='localhost'
+   export PROXYSQL_PORT='6032'
+
+   # PXC admin credentials for connecting to pxc-cluster-node.
+   export CLUSTER_USERNAME='admin'
+   export CLUSTER_PASSWORD='admin'
+   export CLUSTER_HOSTNAME='localhost'
+   export CLUSTER_PORT='3306'
+
+   # proxysql monitoring user. proxysql admin script will create this user in pxc to monitor pxc-nodes.
+   export MONITOR_USERNAME="monitor"
+   export MONITOR_PASSWORD="monit0r"
+   
+   # Application user to connect to pxc-node through proxysql
+   export CLUSTER_APP_USERNAME="proxysql_user"
+   export CLUSTER_APP_PASSWORD="passw0rd"
+   
+   # ProxySQL hostgroup IDs
+   export WRITER_HOSTGROUP_ID='10'
+   export READER_HOSTGROUP_ID='11'
+   export BACKUP_WRITER_HOSTGROUP_ID='12'
+   export OFFLINE_HOSTGROUP_ID='13'
+
+   # ProxySQL read/write configuration mode.
+   export MODE="singlewrite"
+
+   # max_connections default (used only when INSERTing a new mysql_servers entry)
+   export MAX_CONNECTIONS="1000"
+
+   # Determines the maximum number of writesets a node can have queued
+   # before the node is SHUNNED to avoid stale reads.
+   export MAX_TRANSACTIONS_BEHIND=100
+
+   # Connections to the backend servers (from ProxySQL) will use SSL
+   export USE_SSL="no"
+
+   # Determines if a node should be added to the reader hostgroup if it has
+   # been promoted to the writer hostgroup.
+   # If set to 'yes', then all writers (including backup-writers) are added to
+   # the read hostgroup.
+   # If set to 'no', then none of the writers (including backup-writers) are added.
+   # If set to 'backup', then only the backup-writers will be added to
+   # the read hostgroup.
+   export WRITERS_ARE_READERS="backup"
+
+.. _pxc.proxysql.v2.admin-tool:
+
+Running |proxysql-admin| tool
+================================================================================
+
+It is recommended to :ref:`change default ProxySQL credentials
+<default-credentials>` before running ProxySQL in production.  Make sure that
+you provide ProxySQL location and credentials in the configuration file.
+
+Provide superuser credentials for one of the |PXC| nodes.  The
+``proxysql-admin`` script will detect other nodes in the cluster automatically.
+
+.. contents::
+   :local:
+
+.. _pxc.proxysql.v2.admin-tool.enable:
+
+--enable
+--------------------------------------------------------------------------------
+
+This option creates the entry for the Galera hostgroups and adds the |PXC| nodes
+to ProxySQL.
+  
+It will also add two new users into the Percona XtraDB Cluster with the USAGE
+privilege; one is for monitoring the cluster nodes through ProxySQL, and another
+is for connecting to the PXC Cluster node via the ProxySQL console.
+  
+.. note::
+
+   Please make sure to use super user credentials from |PXC| to
```

Comment:
Same as above

Reply:
here the example that follows uses sudo which actually makes this note redundant